tests/bluetooth/bsim_bt/compile.sh

```diff
@@ -20,47 +20,49 @@ mkdir -p ${WORK_DIR}
 
 source ${ZEPHYR_BASE}/tests/bluetooth/bsim_bt/compile.source
 
-app=tests/bluetooth/bsim_bt/bsim_test_bond_overwrite_allowed compile
-app=tests/bluetooth/bsim_bt/bsim_test_bond_overwrite_denied compile
-app=tests/bluetooth/bsim_bt/bsim_test_notify compile
-app=tests/bluetooth/bsim_bt/bsim_test_notify_multiple compile
-app=tests/bluetooth/bsim_bt/bsim_test_eatt_notif conf_file=prj.conf compile
-app=tests/bluetooth/bsim_bt/bsim_test_gatt_caching compile
-app=tests/bluetooth/bsim_bt/bsim_test_eatt conf_file=prj_collision.conf compile
-app=tests/bluetooth/bsim_bt/bsim_test_eatt conf_file=prj_multiple_conn.conf compile
-app=tests/bluetooth/bsim_bt/bsim_test_eatt conf_file=prj_autoconnect.conf compile
+app=tests/bluetooth/bsim_bt/bsim_test_bond_overwrite_allowed compile &
```

Comment:
Maybe a silly question: isn't the build itself mostly parallel already? did you see an improvement from this?

Reply:
The actual build step (ninja) is sufficiently parallelised, but the configuration step (cmake) is not. Doing this reduces the build time by approx. 5 minutes.